packages/airnode-validator/src/ois/ois.ts

```diff
@@ -22,20 +21,17 @@ export const operationParameterSchema = z.object({
 
 export const fixedParameterSchema = z.object({
   operationParameter: operationParameterSchema,
-  // TODO: This could be any JSON value
   value: z.string(),
 });
 
 export const methodSchema = z.union([z.literal('get'), z.literal('post')]);
 
 export const endpointOperationSchema = z.object({
   method: methodSchema,
-  // TODO: Validate URL path
   path: z.string(),
 });
 
 export const endpointParameterSchema = z.object({
-  // TODO: Endpoint name validation
```

Comment:
This just need to be unuqie in OIS. I'll include this into some issue (not sure which).